sdk/communication/Azure.Communication.CallingServer/tests/CallingServerClient/CallingServerClientTests.cs

```diff
@@ -91,6 +93,33 @@ public void JoinCall_Returns404NotFound(string serverCallId, CommunicationIdenti
             Assert.AreEqual(ex?.Status, 404);
         }
 
+        [Test]
+        public void DeleteRecording_Returns200Ok()
+        {
+            CallingServerClient callingServerClient = CreateMockCallingServerClient(200);
+            var response = callingServerClient.DeleteRecording(new Uri(AmsDeleteUrl));
+            Assert.AreEqual((int)HttpStatusCode.OK, response.Status);
+        }
```

Comment:
Can we also add an integration test here ? Like deleting the recording and then try to get it to check if its actually deleted.

Reply:
I was following convention set by the download APIs which work in a similar way. I think this is difficult to test without user intervention because of the need to create a recording in the storage service, and there is no way to check status in that manner. Instead we can rely on the existing storage integration test and test end-to-end with our test runners.

@cochi2 is my thinking here correct or was there a way we can add integration tests in these download/delete scenarios?